File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void){

    int coins = 0;
    float change;
    do{
        change = get_float("Change owed: ");
    }while(change<=0);

    int cents = round(change * 100);
    printf("Change and converted: %f, %i\n", change, cents);   // converting change to cents


    while(cents!=0){
        while(cents>=25){
            cents-=25;
            coins++;
        }

        while(cents>=10){
            cents-=10;
            coins++;
        }


        while(cents>=5){
            cents-=5;
            coins++;
        }


        while(cents>=1){
            cents-=1;
            coins++;
        }

    }

    printf("Final coins needed is:%i\n", coins);


}

File: pset6/cash/cash.py
from cs50 import get_float

num = 0

while True:
    n=get_float("Change owed: ")
    if n > 0:
        while n >= 0.25:
            num += 1
            n = round(n-0.25, 2)
        while n >= 0.10:
            num += 1
            n = round(n-0.10, 2)
        while n >= 0.05:
            num += 1
            n = round(n-0.05, 2)
        while n >= 0.01:
            num += 1
            n = round(n-0.01, 2)
        print(num)
        break